data/interfaces/default/graphs.html

```diff
@@ -232,6 +241,74 @@
+                    <div class="col-md-12">
+                        <h4><i class="fa fa-calendar"></i> Data added <small>Last 12 months</small></h4>
+                        <p class="help-block">
+                            The combined total of tv, movies, and music by month.
+                        </p>
+                        <div class="graphs-instance">
+                            <div class="watch-chart" id="chart_div_data_by_month">
+                                <div class="graphs-load"><i class="fa fa-refresh fa-spin"></i> Loading chart...
+                                </div>
+                                <br>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-6">
+                        <h4><i class="fa fa-calendar"></i> <span class="yaxis-text">Added</span> by day of week <small>Last <span class="days">30</span> days</small></h4>
+                        <p class="help-block">
+                            The combined total of tv, movies, and music added by per day of the week.
+                        </p>
+                        <div class="graphs-instance">
+                            <div class="watch-chart" id="chart_div_data_by_dayofweek" style="float: left;">
+                                <div class="graphs-load">
+                                    <i class="fa fa-refresh fa-spin"></i> Loading chart...
+                                </div>
+                                <br>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <h4><i class="fa fa-clock-o"></i> <span class="yaxis-text">Added</span> by hour of day <small>Last <span class="days">30</span> days</small></h4>
+                        <p class="help-block">
+                            The combined total of tv, movies, and music added per hour of the day.
+                        </p>
+                        <div class="graphs-instance">
+                            <div class="watch-chart" id="chart_div_data_by_hourofday">
+                                <div class="graphs-load">
+                                    <i class="fa fa-refresh fa-spin"></i> Loading chart...
+                                </div>
+                                <br>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-12">
+                        <h4><i class="fa fa-calendar"></i> <span class="yaxis-text">Added</span> by day<small> Last <span class="days">30</span> days</small></h4>
+                            <p class="help-block">
+                                The combined total of tv, movies, and music added by per day of the week.
```

Comment:
... per day. Click a graph point to open up a list of items played for that specific date.